presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AggregationNode.java

```diff
@@ -82,6 +86,14 @@ public AggregationNode(
                 .map(Aggregation::getOutputSymbol)
                 .forEach(outputSymbols::add);
         this.outputSymbols = outputSymbols.build();
+
+        ImmutableSet.Builder<Symbol> inputSymbols = ImmutableSet.builder();
+        inputSymbols.addAll(getGroupingKeys());
+        hashSymbol.ifPresent(inputSymbols::add);
+        aggregations.stream()
+                .map(Aggregation::getInputSymbols)
+                .forEach(inputSymbols::addAll);
+        this.inputSymbols = inputSymbols.build();
```

Comment:
@dain Should we include `groupIdSymbol` into `inputSymbols`?

Reply:
I believe `getGroupingKeys()` above includes it